File: DevExpress-Demo/CS/MVVMDemo.Wpf/Modules/POCOViewModels/DependentPropertiesViewModel.cs
using DevExpress.Mvvm.DataAnnotations;

namespace MVVMDemo.POCOViewModels {
    public class DependentPropertiesViewModel {
        public virtual string FirstName { get; set; }

        public virtual string LastName { get; set; }

        [DependsOnProperties("FirstName", "LastName")]
        public string FullName { get { return FirstName + " " + LastName; } }
    }
}

File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/POCOViewModels/DependentPropertiesViewModel.vb
Imports DevExpress.Mvvm.DataAnnotations

Namespace MVVMDemo.POCOViewModels

    Public Class DependentPropertiesViewModel

        Public Overridable Property FirstName As String

        Public Overridable Property LastName As String

        <DependsOnProperties("FirstName", "LastName")>
        Public ReadOnly Property FullName As String
            Get
                Return FirstName & " " & LastName
            End Get
        End Property
    End Class
End Namespace
